src/common/cockpitwebcertificate.c

```diff
@@ -64,7 +63,7 @@ load_cert_from_dir (const char *dir_name,
   if (n < 0)
     {
       if (errno != ENOENT)
-        asprintfx (error, "Error loading certificates from %s: %m", dir_name);
+        err (EXIT_FAILURE, "Error loading certificates from %s", dir_name);
```

Comment:
Also permissions errors, which is what I was actually thinking about.

The real thing I'm trying to solve here is to split the `ENOENT` case out from the rest of them, while not losing the nicely formatted error message in the `ENOENT` case.

But I'm actually not 100% sure about this change anymore.  I'll need to think it over, because it kinda conflicts with the SIGHUP stuff: in case certificate loading fails on SIGHUP then it's obviously correct to continue to use the old certificate, not abort the entire program.